app/Http/Controllers/Api/DepartmentsController.php

```diff
@@ -15,7 +15,7 @@ class DepartmentsController extends Controller
     /**
      * Display a listing of the resource.
      *
-     * @author [A. Gianotto] [<snipe@snipe.net>]
```

Comment:
maybe lets add another @author here, as the original author hasn't gone away

Reply:
@Godmartinz Did you maybe remove me from the wrong method? 

You'd add yourself to the new `update()` method, since you wrote that one.

Reply:
oops! i goofed. corrected now